### PR TITLE
Update Snapshots page

### PR DESCRIPTION
**Pull request by winkerVSbecks**
Reframing the documentation to become more agnostic (eg: use "test" instead of "story")
Explain what a test is how a story and E2E test are snapshotted
Explain when a snapshot is captured

https://linear.app/chromaui/issue/CHDX-960/snapshots
https://linear.app/chromaui/issue/CHDX-569/explain-when-snapshot-is-taken

### CODE REVIEW

**Review by domyen — APPROVED** [review]
LGTM! For the diagrams, perhaps we can use Mermaid instead for maintainability (not for aesthetics)?

File: src/content/troubleshooting/faq/what-is-archive.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+sidebar: { hide: true }
+title: What is an archive?
+section: "uiTestsAndReview"
+---
+
+# What is an archive?
+
+An **archive** is a self-contained, re-renderable HTML “snapshot” of your UI captured during Playwright or Cypress tests. It contains all the assets and data required to re-render your app UI for Chromatic’s [**Visual Tests**](/docs/test) and [**UI Review**](/docs/review) features.
+
+The archives generated during each test run can be accessed through the Chromatic web app and are fully inspectable, enabling you to troubleshoot changes and errors effectively.
```
Comment (domyen) [inline src/content/troubleshooting/faq/what-is-archive.mdx:12]:
```suggestion
The archives generated during each test run can be accessed through the Chromatic web app and are fully inspectable, enabling you to troubleshoot changes and errors in your own browser (without needing to run Playwright or Cypress again).
```

**Review by domyen — APPROVED** [review]

**Comment by winkerVSbecks** [comment]
@domyen I tried that first. Mermaid doesn't have a great option for this type of diagram unfortunately.

File: src/images/snapshot-with-interactions-and-delay.png
Comment (cgbl-90) [inline src/images/snapshot-with-interactions-and-delay.png:1]: OMG! love this!

**Review by cgbl-90 — APPROVED** [review]
love these changes

**Pull request merged**